client/pages/backend.tsx

```diff
@@ -14,8 +15,7 @@ const Backend = () => {
       setImg(`data:image/jpeg;base64,${data}`);
     };
 
-    // TODO: replace with proper type
-    const onApiResonse = (data: any) => {
+    const onApiResonse = (data: ResponseData) => {
```

Comment:
if the api returns "no person found" for example, does the parsing function return just a string or an obj?
we might need to add `data; ResponseData | string` to handle both cases?

Reply:
yeah i think you are right. do you mind helping me fix that? I'm trying to figure out some git issues might take sometime. tysm!

Reply:
thanks for catching, changed!